django/contrib/auth/__init__.py

```diff
@@ -205,6 +205,11 @@ def get_user(request):
     If no user is retrieved, return an instance of `AnonymousUser`.
     """
     from .models import AnonymousUser
+    assert hasattr(request, 'session'), (
+        "The Django authentication app must be installed to use this feature."
```

Comment:
Shouldn't that talk about the session app? After all the assert checks for `request.session`